packages/rsbuild-plugin-angular/src/lib/config/create-config.ts

```diff
@@ -21,7 +21,7 @@ export function createConfig(
     '@angular/platform-server/init',
   ];
 
-  const isRunningDevServer = process.argv.splice(2)[0] === 'dev';
```

Comment:
I replaced `process.argv.splice(2)[0]` with the immutable version `process.argv.at(2)`. This is in general best practice but was also causing testing issues.